packages/flutter_tools/lib/src/test/coverage_collector.dart

```diff
@@ -40,7 +40,7 @@ class CoverageCollector {
     });
 
     printTrace('pid $pid (port $port): collecting coverage data...');
-    final Map<dynamic, dynamic> data = await collect(host.address, port, false, false);
+    final Map<String, dynamic> data = await collect(host.address, port, false, false);
```

Comment:
It is. I've also updated the coverage package to make it explicit that the map returned is a Map<String, dynamic>
https://codereview.chromium.org/2670943002/